lint/analyzers_test.go

```diff
@@ -42,6 +44,18 @@ func testAnalyzers(t *testing.T, code string, analyzers ...*analysis.Analyzer) [
 		nil,
 	)
 
+	if suppressParsingCheckingErrors {
+		// Suppress parser errors
+		config.HandleParserError = func(err analysis.ParsingCheckingError, program *ast.Program) error {
+			return nil
+		}
+
+		// Suppress checker errors
+		config.HandleCheckerError = func(err analysis.ParsingCheckingError, checker *sema.Checker) error {
+			return nil
+		}
```

Comment:
Maybe instead of ignoring the errors, maybe assert that expected errors were reported, e.g. by just collecting each parser error and checker error, and then asserting in the test function that calls the `testAnalyzers` helper,  that certain errors occurred (I guess none?).

Reply:
sounds good, will do this